client/lib/purchases/index.js

```diff
@@ -92,20 +92,6 @@ function handleRenewNowClick( purchase, siteSlug ) {
 		product_slug: purchase.productSlug,
 	} );
 
-	if ( hasPrivacyProtection( purchase ) ) {
```

Comment:
This might be problematic? I'm _guessing_ that this codepath might be triggered if someone visits from a direct renew link. And if we're not fast enough with the backend changes, their privacy sub might expire as a result (even though they've paid for domain renewal) and privacy protection will be off (and for WWD domains in non-GDPR countries that's actually bad).

A backend change first that would _not_ disable privacy protection when the privacy subscription is removed due to expiration could possibly prevent this problem - what do you think? 🤔

Reply:
Yep. Will remove disabling privacy on `deactivate` on the back-end. 👍 It can be a follow up diff.